src/css/textarea.less

```diff
@@ -6,6 +6,11 @@
 
     // TODO: why is this here?
     .user-select(text);
+
+   > textarea {
+     opacity: 0; // Defense against iOS cursor appearing in client pages
```

Comment:
Is this to fix #310? Should be fixed already (https://github.com/mathquill/mathquill/pull/554/files#diff-f00a1908060a90535e10dbd689e744ddR22)